app/views/service_mailer/notify_owner_email.text.erb

```diff
@@ -1,11 +1,11 @@
 # Your changes to <%= @service.name %> have been approved.
 
 
-<% if @service.publicly_visible? %>
+<% if @service.publicly_visible? && ENV['SCOUT_URL'] %>
 
-They're now publicly visible in the Family Information Service directory:
+They are now publicly visible in the directory:
 
-https://directory.familyinfo.buckinghamshire.gov.uk/service/<%= @service.id %>
+<%= ENV['SCOUT_URL'] %>/service/<%= @service.id %>
```

Comment:
There are currently two scout URLs potentially more. Thinking about it how does this system know which directory the service has been updated on - since it could be shown on both 🤔